paddle/fluid/operators/CMakeLists.txt

```diff
@@ -171,6 +171,7 @@ endif()
 if (WITH_CINN)
   op_library(cinn_launch_op SRCS cinn_launch_op.cc cinn_launch_op.cu.cc DEPS transform_desc cinn_compiler cinn ${OP_HEADER_DEPS})
   cc_test(cinn_launch_op_test SRCS cinn_launch_op_test.cc DEPS cinn_compiler cinn_launch_op elementwise_add_op)
+  set_tests_properties(cinn_launch_op_test PROPERTIES ENVIRONMENT OMP_NUM_THREADS=1)
```

Comment:
This comment is FYI (for you information). Currently, this test failed in Paddle-CINN CI due to libcublas version. Thank you for not setting it as RUN_TYPE=CINN

Reply:
Got it